SDK Showcase/SDK Showcase/Model/BrowserViewModel.swift

```diff
@@ -0,0 +1,90 @@
+//  Copyright © 2025 Onewelcome Mobile Identity. All rights reserved.
+
+import SwiftUI
+import WebKit
+
+class BrowserViewModel: NSObject, ObservableObject {
+    var browserInteractor: BrowserRegistrationInteractor {
+        @Injected var interactors: Interactors
+        return interactors.browserInteractor
+    }
+    
+    weak var webView: WKWebView? {
+        didSet {
+            webView?.navigationDelegate = self
+        }
+    }
+
+    @Published var urlString = ""
+    @Published var rediredURL: URL?
+    @Published var canGoBack = false
+    @Published var canGoForward = false
+    @Published var isFinished = false
+    @Published var errorMessage = ""
+    
+    init(url: URL) {
+        self.urlString = url.absoluteString
+    }
+
+    func loadURLString() {
+        if let url = URL(string: urlString) {
```

Comment:
`guard`? `flatMap`? :)

Reply:
guard is not a replacement for if. I use it when it make sense